boards/weio/board.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_weio
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the WeIO board
+ *
+ * @author      Paul RATHGEB <paul.rathgeb@skynet.be>
+ *
+ * @}
+ */
+
+#include "board.h"
+
+static void leds_init(void);
+extern void SystemInit(void);
+
+void board_init(void)
+{
+    /* initialize core clocks via CMSIS function */
+    SystemInit();
+    /* initialize the CPU */
+    cpu_init();
+    /* initialize the boards LEDs */
+    leds_init();
+}
+
+/**
+ * @brief Initialize the boards on-board LEDs (LED1 to LED4)
+ *
+ * The LED initialization is hard-coded in this function. As the LEDs are soldered
+ * onto the board they are fixed to their CPU pins.
+ *
+ * The LEDs are connected to the following pins:
+ * - LED1: P1.18
+ * - LED2: P1.19
+ * - LED3: P1.20
```

Comment:
Are these correct? In `board.h` `BIT13, BIT14, BIT15` are used to define `LED?_PIN`.